p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -160,10 +160,20 @@ Layout Description
         <ecc/>
     </section>
     <section>
-        <description>Hostboot Data (1.125M)</description>
-        <eyeCatch>HBD</eyeCatch>
+        <description>Hostboot Data - READ-ONLY (~819KB)</description>
+        <eyeCatch>HBD_RO</eyeCatch>
         <physicalOffset>0x305000</physicalOffset>
-        <physicalRegionSize>0x120000</physicalRegionSize>
+        <physicalRegionSize>0xCD000</physicalRegionSize>
+        <side>A</side>
+        <sha512Version/>
+        <readOnly/>
+        <ecc/>
+    </section>
+    <section>
+        <description>Hostboot Data - Read-Write (~333KB)</description>
+        <eyeCatch>HBD_RW</eyeCatch>
+        <physicalOffset>0x3D2000</physicalOffset>
+        <physicalRegionSize>0x53000</physicalRegionSize>
```

Comment:
in channel you state that RW size is like 128k.  128k = 0x20000 ..  ECC pad lifts to around 0x24000.

so this # seems over double .

Reply:
The total amount of both seems correct though

Reply:
Good catch.  The bug was that I calculated 0x128 x 1KB and now 128 x 1KB.  Next patchset should show the correct values.